### ISSUE TITLE
Have the ability for links to be attached by edition

### ISSUE THREAD

**Issue by BrittanyBunk** [internetarchive/openlibrary#2455]
Right now, the links added in only go under a work, but it'd be great to have edition-specific links be able to go with the editions they belong with.

**Comment by mekarpeles** [comment]
Is this true? Links are addable to editions and they should show up on the UI.
![openlibrary org_books_OL24461477M_The_autobiography_of_Fidel_Castro_edit](https://user-images.githubusercontent.com/978325/67733315-426ab200-f9bb-11e9-8026-4b2d80c5af4d.png)

**Comment by BrittanyBunk** [comment]
I said for individual editions, not works. Unless it's labeled by edition, they all show up in the same place @mekarpeles this shouldn't've been closed. It's not addressed.

**Comment by cdrini** [comment]
I think this is a valid request. I often want to add a link to the publisher's page for the editiong (e.g. https://www.cambridge.org/us/academic/subjects/computer-science/algorithmics-complexity-computer-algebra-and-computational-g/probability-and-computing-randomization-and-probabilistic-techniques-algorithms-and-data-analysis-2nd-edition?format=AR&isbn=9781107154889 ). These are edition-specific and don't apply to the work

**Comment by BrittanyBunk** [comment]
@cdrini thank you. I appreciate you understanding this.

**Comment by guyjeangilles** [comment]
Agreed that this is a valid request. Under the `Specific To This Edition` section is there is no unique for adding links. Adding them via the description is a work around but not a proper solution.
